ansible_collections/arista/avd/roles/eos_validate_state/tasks/main.yml

```diff
@@ -0,0 +1,281 @@
+        commands: "show mlag detail | json"
+        waitfor:
+          - "result[0].state eq 'active'"
+          - "result[0].negStatus eq 'connected'"
+        retries: 2
+        interval: 2
+      ignore_errors: "{{ validation_mode_loose }}"
+      when: |
+        (mlag_configuration is defined and mlag_configuration is not none)
+      tags:
+        - avd
+
+    - name: Gather IP reachability state (directly connected interfaces)
+      eos_command:
+        commands: "ping {{ hostvars[ethernet_interface.value.peer]['ethernet_interfaces'][ethernet_interface.value.peer_interface]['ip_address'] | ipaddr('address')}} source {{ ethernet_interface.value.ip_address  | ipaddr('address')}} repeat 1"
+      loop: "{{ethernet_interfaces | default({}) | dict2items}}"
+      loop_control:
+        loop_var: ethernet_interface
+      ignore_errors: "{{ validation_mode_loose }}"
+      when: |
+        (ethernet_interfaces is defined and ethernet_interfaces is not none) and
+        (ethernet_interface.value.type is defined and ethernet_interface.value.type is not none and ethernet_interface.value.type == 'routed') and
+        (ethernet_interface.value.ip_address is defined and ethernet_interface.value.ip_address is not none) and
+        (ethernet_interface.value.peer is defined and ethernet_interface.value.peer is not none) and
+        (ethernet_interface.value.peer_interface is defined and ethernet_interface.value.peer_interface is not none) and
+        (hostvars[ethernet_interface.value.peer]['ethernet_interfaces'][ethernet_interface.value.peer_interface]['ip_address'] is defined and
+        hostvars[ethernet_interface.value.peer]['ethernet_interfaces'][ethernet_interface.value.peer_interface]['ip_address'] is not none)
+      register: ip_reachability_state
+      tags:
+        - ip_reachability
+
+    - name: validate ip reachability (directly connected interfaces)
+      assert:
+        that:
+          - ip_reachability_test.stdout[0] | regex_search("1 received")
+        quiet: true
+      loop: "{{ ip_reachability_state.results }}"
+      loop_control:
+        loop_var: ip_reachability_test
+      ignore_errors: "{{ validation_mode_loose }}"
+      when: ( ip_reachability_test.skipped is not defined )
+      register: ip_reachability_results
+      tags:
+        - ip_reachability
+
+    - name: Gathter ip route summary and ArBGP state
+      eos_command:
+        commands: "show ip route summary"
+      register: ip_route_summary
+      tags:
+        - ArBGP_state
+
+    - name: validate ArBGP is configured and operating
+      assert:
+        that:
+          - ip_route_summary.stdout_lines[0][0] | regex_search("multi-agent")
+          - ip_route_summary.stdout_lines[0][1] | regex_search("multi-agent")
+        quiet: yes
+      when: ( service_routing_protocols_model is defined and service_routing_protocols_model == "multi-agent" )
+      ignore_errors: "{{ validation_mode_loose }}"
+      tags:
+        - ArBGP_state
```

Comment:
Only lowercase should be used to reduce risk of confusion.

Reply:
agreed!